pylintrc

```diff
@@ -179,4 +180,4 @@ int-import-graph =
 [EXCEPTIONS]
 overgeneral-exceptions = Exception
 
-# 3dd01da1adbb2cac2fbed3cbc578b3f647b792c7
```

Comment:
Cool. :+1:

On Fri, Dec 18, 2015 at 10:07 AM, Ned Batchelder <notifications@github.com>
wrote:

> In pylintrc <https://github.com/edx/edx-lint/pull/30#discussion_r48033146>
> :
>
> > @@ -179,4 +180,4 @@ int-import-graph =
> >  [EXCEPTIONS]
> >  overgeneral-exceptions = Exception
> >
> > -# 3dd01da1adbb2cac2fbed3cbc578b3f647b792c7
>
> Maybe I should add a comment here: "edx-lint write" includes a hash at the
> end of the file so that edits to the generated file will be detected in the
> future to prevent losing edits. The hash will change any time the file
> contents change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-lint/pull/30/files#r48033146>.
>